ceph/rbd/workflows/rbd.py

```diff
@@ -476,29 +481,38 @@ def remove_single_image_and_verify(**kw):
             "rbd": <>,
             "pool": <>,
             "image": <>,
+            "namespace":<>,
             "test_ops_parallely": <>
         }
     """
     rbd = kw.get("rbd")
     pool = kw.get("pool")
     image = kw.get("image")
+    namespace = kw.get("namespace", None)
     test_ops_parallely = kw.get("test_ops_parallely", False)
 
     log.info(f"Removing image {image} from pool {pool}")
-    out, err = rbd.rm(pool=pool, image=image)
+    if namespace is not None:
+        out, err = rbd.rm(pool=pool, namespace=namespace, image=image)
+    else:
+        out, err = rbd.rm(pool=pool, image=image)
     if out or err and "100% complete" not in out + err:
         log.error(
-            f"Image remove failed for image {pool}/{image} with error {out} {err}"
+            f"Image remove failed for image {pool}/{namespace}/{image} with error {out} {err}"
         )
         if test_ops_parallely:
             raise Exception(
-                f"Image remove failed for image {pool}/{image} with error {out} {err}"
+                f"Image remove failed for image {pool}/{namespace}/{image} with error {out} {err}"
             )
         return 1
 
-    rc = list_images_and_verify_with_input(rbd=rbd, pool=pool, images=[image])
+    rc = list_images_and_verify_with_input(
```

Comment:
this might cause some existing tests to fail which doesn't pass `args` with `namespace` so it's good to create new method itself, @manasagowri WDYT?

Reply:
Again, if you watch the complete code in VScode you will see, existing code will not be impacted at all. Here is how

```
namespace = kw.get("namespace", None)
.
.
.

if namespace is not None:
        out, err = rbd.rm(pool=pool, namespace=namespace, image=image)
else:
        out, err = rbd.rm(pool=pool, image=image)
```

so existing methods don't pass namespace and hence its value is None and existing code will be executed.